README.md

```diff
@@ -187,6 +187,7 @@ Further Information
 | elasticsearch_indices_search_query_total                             | counter    | 1           | Total number of queries                                                                             |
 | elasticsearch_indices_segments_count                                 | gauge      | 1           | Count of index segments on this node                                                                |
 | elasticsearch_indices_segments_memory_bytes                          | gauge      | 1           | Current memory size of segments in bytes                                                            |
+| elasticsearch_indices_settings_creation_date                         | gauge      | 1           | Index setting value for index.creation_date                                                         |
```

Comment:
This should be more descriptive, and should match the Help text.

Maybe something like "Timestamp of the index creation".